Feature/COM-3264 Integrated Shipping Method API's

hrishid opened the pull request.

Commit:
COM-3264 Integrated Shipping Method API's

rajeshdatir commented on scripts/modules/b2b-account/edit-quote.js:
```diff
@@ -873,28 +878,92 @@ define([
+        },    
+   
+        getAvailableShippingMethods: function()
+        { var self = this;
```
add a variable on new line

rajeshdatir commented on scripts/modules/b2b-account/edit-quote.js:
```diff
@@ -873,28 +878,92 @@ define([
+                {
+                       "quoteId": self.model.get('id'),
+                       "draft": true 
+                 
```
Remove blank lines

rajeshdatir commented on scripts/modules/b2b-account/edit-quote.js:
```diff
@@ -873,28 +878,92 @@ define([
+        });         
+       },
+       shippingMethodSelectionChanged: function()
+       {    var self = this;              
```
add curly braces after _function() {_. make it consistent with other functions

Commit:
COM-3264 Fixed ShippingMethod

Commit:
COM-3264 Fixed ShippingMethod for the first Initialization.

Commit:
COM-3264 Fixed FulfillmentInfo for Audit History

rakesh-dontula commented on scripts/modules/b2b-account/edit-quote.js:
```diff
@@ -873,28 +878,93 @@ define([
+            var fulfillmentInfo = self.model.get("fulfillmentInfo");
+            if(fulfillmentInfo)
+            {
+            var json = JSON.parse(JSON.stringify(
```
indent properly. Use some formatter to format the code properly.

Commit:
COM-3264 Added Refresh Logic and Fixed Buttons on modal

Commit:
COM-3264 Formatted It by using visual Studio formatter

Kibo-Sagar committed:
COM-3264: Added a required validations for quote before submitting.

Added a following required validations (COM-3367)-
Required fields:
    Quote Name (not null or empty string)
    Created By (not null)
    Quote Items (at least 1 item)
    Shipping info (if any direct ship items)
Quotes State -
    Pending (Submit for Approval)
    In Review (Approve Quote)
    Ready for Checkout (Submit for Approval)

Commit:
COM-3264 Fixed Shipping Method Width

Commit:
COM-3264 Fixed Shipping Methods

Kibo-Sagar committed:
COM-3264: Added Expiration Date validation

Commit:
COM-3264 Fixed GetAvailableShippingMethods

Commit:
Code review changes

ghost commented:
As part of my rebase I reverted a no-op change to `templates/modules/multi-ship-checkout/shipping-method-item.hypr.live` as it wasn't needed. Cleaned up some whitespace/formatting issues as well. No functional changes.
